src/Kafka/Settings/Settings.tsx

```diff
@@ -0,0 +1,148 @@
+import {
+  Alert,
+  AlertActionCloseButton,
+  AlertGroup,
+  AlertVariant,
+  Button,
+  Card,
+  CardBody,
+  Flex,
+  FlexItem,
+  Level,
+  LevelItem,
+  Modal,
+  ModalVariant,
+  Page,
+  PageSection,
+  Spinner,
+  Switch,
+  Title,
+} from "@patternfly/react-core";
+import type { AlertProps } from "@rhoas/app-services-ui-shared";
+import type { FunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+
+export type SettingsProps = {
+  showAlert: AlertProps[];
+  isChecked: boolean;
+  isModalOpen: boolean;
+  onChange: () => void;
+  onDisable: () => void;
+  onClose: () => void;
+  closeAlertAction: (key: string | undefined) => void;
+  isLoading: boolean;
+};
+
+export const Settings: FunctionComponent<SettingsProps> = ({
+  showAlert,
+  isChecked,
+  onChange,
+  isModalOpen,
+  onDisable,
+  onClose,
+  closeAlertAction,
+  isLoading,
+}) => {
+  const { t } = useTranslation("kafka");
+
+  return (
+    <>
+      <PageSection>
+        <Card>
+          <CardBody>{t("settings.kafka_instance_settings_label")}</CardBody>
+        </Card>
+      </PageSection>
+      <Page>
+        <PageSection isWidthLimited isCenterAligned>
+          <Card>
+            <CardBody>
+              <Level>
+                <LevelItem>
+                  <Flex>
+                    <FlexItem spacer={{ default: "spacerNone" }}>
+                      <Title headingLevel={"h3"}>
+                        {t("settings.connection_re_authentication_label")}
+                      </Title>
+                    </FlexItem>
+                    <FlexItem>
+                      {isChecked ? (
+                        "On"
+                      ) : isLoading ? (
+                        <>
+                          <Spinner
+                            size="md"
+                            aria-valuetext={"Turning off..."}
+                            aria-label={"Turning off..."}
+                          />{" "}
+                          {"Turning off..."}
+                        </>
+                      ) : (
+                        "Off"
+                      )}
+                    </FlexItem>
+                  </Flex>
+                </LevelItem>
+                <LevelItem>
+                  <Switch
+                    id="Connection-re-authentication-switch"
+                    isChecked={isChecked}
+                    onChange={onChange}
```

Comment:
Please add `aria-label` here. It should have the text string `connection_re_authentication_label` as the value.